tests/scripts/all.sh

```diff
@@ -2166,16 +2228,12 @@ component_test_psa_crypto_config_accel_ffdh () {
     # Algorithms and key types to accelerate
     loc_accel_list="ALG_FFDH KEY_TYPE_DH_KEY_PAIR KEY_TYPE_DH_PUBLIC_KEY"
 
+    helper_libtestdriver1_adjust_config
+
     # Configure and build the test driver library
     # -------------------------------------------
 
-    # Disable ALG_STREAM_CIPHER and ALG_ECB_NO_PADDING to avoid having
-    # partial support for cipher operations in the driver test library.
-    scripts/config.py -f include/psa/crypto_config.h unset PSA_WANT_ALG_STREAM_CIPHER
-    scripts/config.py -f include/psa/crypto_config.h unset PSA_WANT_ALG_ECB_NO_PADDING
-
-    loc_accel_flags=$( echo "$loc_accel_list" | sed 's/[^ ]* */-DLIBTESTDRIVER1_MBEDTLS_PSA_ACCEL_&/g' )
-    make -C tests libtestdriver1.a CFLAGS=" $ASAN_CFLAGS $loc_accel_flags" LDFLAGS="$ASAN_CFLAGS"
+    helper_libtestdriver1_make_drivers "$loc_extra_list"
```

Comment:
`$loc_extra_list` is undefined here (found by CI).
```suggestion
    helper_libtestdriver1_make_drivers "$loc_accel_list"
```

Reply:
Aw, I'm sure I remember fixing that locally, must have pushed the wrong version, sorry.

Reply:
Ah, was fixed in my local copy but not committed... :/